satellite/repair/repairer/repairer.go

```diff
@@ -109,13 +109,17 @@ func (service *Service) worker(ctx context.Context, seg *pb.InjuredSegment) (err
 
 	workerStartTime := time.Now().UTC()
 
-	zap.L().Info("Limiter running repair on segment", zap.Binary("segment", seg.GetPath()))
+	service.log.Info("Limiter running repair on segment", zap.Binary("segment", seg.GetPath()))
 	err = service.repairer.Repair(ctx, string(seg.GetPath()))
 	if err != nil {
+		if IrreparableError.Has(err) {
+			service.log.Error("deleting irreparable segment from the queue:", zap.Error(service.queue.Delete(ctx, seg)), zap.Binary("segment", seg.GetPath()))
```

Comment:
could you split this into multiple lines?

Reply:
done